code/src/components/Header.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { useSelector } from 'react-redux';
+import styled from 'styled-components/macro'
+import { OuterWrapper, InnerWrapper } from 'utils/GlobalStyle'
+import mainIcon from '../assets/mainIcon.png'
+
+const Header = () => {
+  // Shows total number of tasks in the list
+  const taskList = useSelector((store) => store.tasks.items);
+
+  // Show number of completed tasks
+  const completedTasks = taskList.filter((task) => task.isCompleted);
+
+  // Returns title and task counter
+  // header1 is white div, header2 is blue div
+  return (
+    <>
+      <OuterWrapper header1>
+        <InnerWrapper>
+          <AppTitle><TodoIcon src={mainIcon} alt="Todo icon" /> Daily Tasks <TodoIcon src={mainIcon} alt="Todo icon" /></AppTitle>
+        </InnerWrapper>
+      </OuterWrapper>
+      <OuterWrapper header2>
+        <InnerWrapper headerInner2>
```

Comment:
I really need to look into this props and using styled components as in this weeks video! You were before your time! Go you! ;)